apps/rd/rd-commondata-dataload/demo-01.yaml

```diff
@@ -5,6 +5,6 @@ metadata:
 spec:
   values:
     job:
-      schedule: "15 17 * * *"
+      schedule: "15 10 * * *"
```

Comment:
10:15 or 11:15?

Reply:
According to the KT this is in UTC so because we are in BST it will run +1 hour.